### DISCUSSION TITLE
How to select an asset group for a job

### DISCUSSION THREAD

**Discussion by bendnorman** [dagster-io/dagster#13792] [Q&A] [answered]
[PUDL](https://github.com/catalyst-cooperative/pudl) has a few pre-configured jobs and about a dozen asset groups. For example, we have a job that runs the ETL for all available years called `etl_full`  and another job that runs the ETL for the most recent year of data called `etl_fast`:

<img width="1505" alt="image" src="https://user-images.githubusercontent.com/17532695/233229112-5e264344-8ea6-4a22-bee8-04b5326d2907.png">

When I click on an asset group and select "Materialize All", dagster uses the default configuration. **Is it possible to materialize asset groups for specific jobs?** For example, it would be nice to be able to execute one asset group like `clean_eia860` using the configuration of the `etl_fast` job.

**Comment by sryza** [comment, marked as answer]
This isn't currently possible. Something you could consider here is using a resource to house the "fast" vs. "full" configuration option, and using that in all your assets to determine what action to take.

Then, when materializing an asset group, you could just supply a single configuration value to control that behavior.

**Reply by bendnorman** [reply]
Got it. Any plans to allow users to filter assets in jobs using asset groups? Something like this:
![image](https://user-images.githubusercontent.com/17532695/233473377-5ec5959b-130b-4f08-9eb4-41a10a413785.png)
so users can execute asset groups of preconfigured jobs?

**Reply by sryza** [reply]
I do think we'd like to enable that - we just need to figure out what the right syntax for it be. Would you be up for filing an issue for this (i.e. filtering asset graphs by group name)? Then I can pass it on to our UI folks.

**Reply by bendnorman** [reply]
Sounds good! Just created #13818
